plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/HiveS3Config.java

```diff
@@ -81,6 +81,7 @@
     private boolean s3preemptiveBasicProxyAuth;
     private String s3StsEndpoint;
     private String s3StsRegion;
+    private boolean anonymousRequestsEnabled;
```

Comment:
Please fix getter and setter as well. 
```suggestion
    private boolean s3AnonymousRequestsEnabled;
```